models/LineCont.yaml

```diff
@@ -0,0 +1,7 @@
+type: LineCont  # comment or string or block
```

Comment:
what are the `type`s of line continuations ?

Reply:
done